src/Tizen.NUI/src/public/Layouting/LayoutItem.cs

```diff
@@ -600,23 +600,20 @@ private bool SetFrame(float left, float top, float right, float bottom, bool ind
                 }
                 else
                 {
-                    if (Owner.Position != null)
+                    if (independent)
```

Comment:
It is added by this patch : https://github.com/Samsung/TizenFX/pull/2210/files#diff-7f6a6b6c3dd860842f5d89bcf00c8d5746b0bbd1554dd6611534cd6461738bd5R561
Is there any svace problem with this fix?

Reply:
Svace warning is reported because of `Owner.Position.Z`(https://github.com/Samsung/TizenFX/pull/2210/files#diff-7f6a6b6c3dd860842f5d89bcf00c8d5746b0bbd1554dd6611534cd6461738bd5R563).
However, `Owner.Position.Z` was removed here(https://github.com/Samsung/TizenFX/pull/2187/files#diff-7f6a6b6c3dd860842f5d89bcf00c8d5746b0bbd1554dd6611534cd6461738bd5L563).
So, i think we don't need to consider Svace problem anymore.